src/configurationProvider.ts

```diff
@@ -80,10 +81,13 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
                         };
                     });
                     resolve([defaultLaunchConfig, ...launchConfigs]);
-                }, (ex) => {
+                } catch (ex) {
+                    if (ex instanceof utility.JavaExtensionNotActivatedError) {
+                        utility.guideToInstallJavaExtension();
```

Comment:
The check logic may work for the scenario where Java debugger is activated mistakenly. But we also need take care of the normal Java debugging scenario. For example, when Java language server is activating, click F5, the user may receive an error message to guide to install Language Support for Java.

Reply:
I should only pop up error message and guide user to install Java extension **ONLY WHEN** redhat.java is not installed. That is, `vscode.extensions.getExtension(JAVA_EXTENSION_ID)` returns the extension instance.